### PR TITLE
Add Google Map Background

### PR DESCRIPTION
**Pull request by miro**
screen shot 2015-03-18 at 16 20 41

This is more "experimental one", and needs some design touch before actually merging - but just a quick proof-of-concept that someone can continue, if this seems to be something we want.

Adds Google Maps background with a pin indicating the new office location. There is a small margin at the top of the pin, so it won't go behind the progress bar.

Progress bar wrapper is stylized to be with white background, rounded corners and small box-shadow.

Untested on mobile screen sizes.

### CODE REVIEW

File: dist/js/mapBackground.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  var mapCenterPointPosition = new google.maps.LatLng(
+    officeCoords[0] + 0.002, 
+    officeCoords[1]
+  );
+
+  var mapOptions = {
+      zoom: 16, // initialize zoom level - the max value is 21
+      streetViewControl: false, // hide the yellow Street View pegman
+      scaleControl: false, // disable users from zooming the Google Map
+      mapTypeId: google.maps.MapTypeId.ROADMAP,
```
Comment (ykarikos) [inline dist/js/mapBackground.js:15]: Why not

```
mapTypeId: google.maps.MapTypeId.SATELLITE
```

**Comment by staltz** [comment]
Nice!!! I'll merge this and convert it into Cycle stuff, and then try to draw a line between old and new offices depending on the current move %. Super thanks!

File: dist/js/mapBackground.js (hunk 1 above)
Comment (miro) [inline dist/js/mapBackground.js:15]: That or HYBRID would be perhaps better than the current one?

### Hybrid

screen shot 2015-03-18 at 16 34 13

### Satellite

screen shot 2015-03-18 at 16 34 22

**Comment by ohanhi** [comment]
@staltz Yeah, a simple polyline should do the trick -- but calculating the phase (based on percentage) may be a bit tricky.
https://developers.google.com/maps/documentation/javascript/examples/polyline-simple

**Comment by staltz** [comment]
What do you mean with phase?

**Comment by ohanhi** [comment]
Just how much of the polyline should be drawn.

File: dist/js/mapBackground.js (hunk 1 above)
Comment (ykarikos) [inline dist/js/mapBackground.js:15]: Yeah, `google.maps.MapTypeId.HYBRID` looks good!

**Comment by staltz** [comment]
Well, point1 is `(lngOld, latOld)` the old office, and point2 is `( lngNew * percentage + lngOld *(1 - percentage), latNew * percentage + latOld *(1 - percentage) )`

**Comment by ohanhi** [comment]
Oh, I was thinking more along the lines of an approximate path via publicly accessible roads. :green_heart:

File: dist/js/mapBackground.js (hunk 1 above)
Comment (staltz) [inline dist/js/mapBackground.js:15]: I'm changing code now, and I'll use HYBRID. :)

**Pull request merged**